packages/react-native/lib/didHooks.js

```diff
@@ -142,13 +142,41 @@ export function useDIDManagement() {
     [wallet],
   );
 
+  const exportDID = useCallback(
+    async ({id, password}) => {
+      const existingDoc = await wallet.getDocumentById(id);
+      if (existingDoc) {
+        const correlationDocuments = await wallet.resolveCorrelations(id);
+        const didKeypairDoc = correlationDocuments.find(doc => {
+          return !!(
+            doc &&
+            typeof doc.type === 'string' &&
+            doc.type.startsWith('Ed25519VerificationKey')
```

Comment:
will this prevent us from exporting non-ED25519 DIDs?

Reply:
Yes, it would.

Reply:
Is that what we want to do? I would think we should export any Did regardless of the keys used.